src/main/java/seedu/address/logic/parser/AddAllergyCommandParser.java

```diff
@@ -34,13 +36,13 @@ public AddAllergyCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddAllergyCommand.MESSAGE_USAGE));
         }
         try {
-            Nric nric = ParserUtil.parseNric(argMultimap.getValue(PREFIX_NRIC).get());
-            Set<Tag> Allergies = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+            Nric nric = parseNric(argMultimap.getValue(PREFIX_NRIC).get());
+            Set<Tag> allergies = parseTags(argMultimap.getAllValues(PREFIX_TAG));
```

Comment:
At this point I think can just rename the tag class to allergy haha. lesser confusion

Reply:
Ok. Ill rename the classes if i have time